src/main/java/io/kabanero/api/restricted/GitHubEndpoints.java

```diff
@@ -51,4 +63,43 @@ public User getUserInfo() throws IOException {
         return new UserService(client).getUser();
     }
 
+    @POST
+    @Path("/team/{teamId}/member/{github_username}")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response addTeamMember(@PathParam("teamId") int teamId, @PathParam("github_username") String githubUsername) throws IOException, ApiException, GeneralSecurityException {
+        try{
+            UserProfile userProfile = UserProfileManager.getUserProfile();
+            String token = userProfile.getAccessToken();
+            GitHubClient client = new GitHubClient();
+            client.setOAuth2Token(token);
+            TeamService teamService = new TeamService(client);      
+            teamService.addMember(teamId, githubUsername);
+            //teamService.addMember is a void method so we don't know if the user was added to the team so return 202 instead of 200
+            return Response.status(202).build();
+        }catch(Exception e){
+            LOGGER.log(Level.SEVERE, e.getStackTrace().toString());
+            return Response.status(500).entity(new ResponseMessage("A problem occured attempting to POST /team/{teamId}/member/{github_username}")).build();
+        }
+    }
+
+    @DELETE
+    @Path("/team/{teamId}/member/{github_username}")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response removeTeamMember(@PathParam("teamId") int teamId, @PathParam("github_username") String githubUsername) throws IOException, ApiException, GeneralSecurityException {
+        try{
+            UserProfile userProfile = UserProfileManager.getUserProfile();
+            String token = userProfile.getAccessToken();
+            GitHubClient client = new GitHubClient();
+            client.setOAuth2Token(token);
+            TeamService teamService = new TeamService(client);      
+            teamService.removeMember(teamId, githubUsername);
+            //teamService.removeMember is a void method so we don't know if the user was removed from the team so return 202 instead of 200
+            return Response.status(202).build();
+        }catch(Exception e){
+            LOGGER.log(Level.SEVERE, e.getStackTrace().toString());
```

Comment:
both logs should be like `LOGGER.log(Level.SEVERE, "Failed to remove team member " + githubUsername , e);`

Reply:
by passing the exception into the log as a parameter will give us the more log output, which is better.